src/Shared/LayoutRenderers/AspNetUserClaimLayoutRenderer.cs

```diff
@@ -18,18 +21,22 @@ namespace NLog.Web.LayoutRenderers
     /// </remarks>
     /// <seealso href="https://github.com/NLog/NLog/wiki/AspNet-User-Claim-Layout-Renderer">Documentation on NLog Wiki</seealso>
     [LayoutRenderer("aspnet-user-claim")]
-    public class AspNetUserClaimLayoutRenderer : AspNetLayoutRendererBase
+    public class AspNetUserClaimLayoutRenderer : AspNetLayoutMultiValueRendererBase
     {
         /// <summary>
         /// Key to lookup using <see cref="ClaimsIdentity.FindFirst(string)"/> with fallback to <see cref="ClaimsPrincipal.FindFirst(string)"/>
         /// </summary>
         /// <remarks>
         /// When value is prefixed with "ClaimTypes." (Remember dot) then ít will lookup in well-known claim types from <see cref="ClaimTypes"/>. Ex. ClaimsTypes.Name
         /// </remarks>
-        [RequiredParameter]
         [DefaultParameter]
         public string ClaimType { get; set; }
 
+        /// <summary>
+        /// If this is set to true, then the ClaimType property is ignored and all Claim Types are rendered
+        /// </summary>
+        public bool All { get; set; }
```

Comment:
Was thinking that if ClaimType is null or empty, then it could mean all.

Reply:
Completed